www/include/reporting/dashboard/viewHostLog.php

```diff
@@ -127,7 +128,7 @@
 /*
  * Stats Display for selected host
  */
-if (isset($id) && $id != "NULL") {
+if ($id !== false) {
```

Comment:
merge this contition with the previous one

Reply:
fix with https://github.com/centreon/centreon/pull/8419/commits/23185b9036383d869664a0590531e69abe7f06c2